SimplenoteUITests/Generic.swift

```diff
@@ -0,0 +1,80 @@
+import XCTest
+
+var stepIndex = 0
```

Comment:
Nitpick: this could be `private` so the only way for consumer code to increment the counter is to call `logStep()`

```suggestion
private var stepIndex = 0
```

Reply:
Thanks! Commited suggestion